Dockerfile

```diff
@@ -21,7 +21,7 @@
 #           -e INPUT_REPO_TOKEN=$GITHUB_AUTH_TOKEN \
 #           -e GITHUB_REPOSITORY="ossf/scorecard" \
 #           laurentsimon/scorecard-action:latest
-FROM gcr.io/openssf/scorecard@sha256:1b2798a386a69f35e61de45e84babde0c878fa775b5828ed90109b5714d49962 as base
+FROM gcr.io/openssf/scorecard@sha256:84ccbdbe49fd6a3c91b04c9971ff7af6bb19fd80d24f3bf08070cdfd4296a54f as base
```

Comment:
I think we cut the v4 release at https://github.com/ossf/scorecard/commit/f7b329e8305d11178836bb35ae95a5f7e7cf2292